Package.swift

```diff
@@ -4,7 +4,7 @@ import PackageDescription
 
 let package = Package(
     name: "Ditto",
-    platforms: [ .iOS(.v11), .macOS(.v11) ],
+    platforms: [ .iOS(.v14), .macOS(.v11) ],
```

Comment:
This package manifest is actually copied from [`cocoa/DittoSwiftPackage.template/Package.swift`](https://github.com/getditto/ditto/blob/567d36e97ba72d75a3d1bafab1191b223040db9c/cocoa/DittoSwiftPackage.template/Package.swift#L7) as part of the release process so we should update that.

Reply:
Great call, @phatblat! I landed https://github.com/getditto/ditto/pull/12825.

Reply:
I am leaving this PR open so we can test the change, but we should close it (not merge it) once we're done testing.